runtime/arc.js

```diff
@@ -234,7 +234,11 @@ ${this.activeRecipe.toString()}`;
       context
     });
     // TODO: pass tags through too
-    manifest.stores.forEach(store => arc._registerStore(store, []));
+    manifest.stores.forEach(store => {
+      if (store.constructor.name == 'StorageStub')
```

Comment:
Is this for backward compatibility? When would the `store` not be a `StorageStub`?
probably question for @shans

Reply:
Yes, this comes from @shans #1259.